nvflare/fuel/hci/client/api_spec.py

```diff
@@ -34,6 +34,17 @@ class CommandCtxKey(object):
     META = "meta"
     CUSTOM_PROPS = "custom_props"
     BYTES_RECEIVER = "bytes_receiver"
+    BYTES_SENDER = "bytes_sender"
+
+
+class SendBytesToServer:
+    def send(self, sock, meta: str):
+        pass
+
+
+class ReceiveBytesFromServer:
+    def receive(self, sock):
+        pass
```

Comment:
suggest either remove these two classes or make them ABC with abstractmethod.

Reply:
We can't remove them.